config/initializers/devise.rb

```diff
@@ -256,7 +258,15 @@
   # ==> OmniAuth
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
-  config.omniauth :github, ENV["GITHUB_APP_ID"], ENV["GITHUB_APP_SECRET"], scope: "user:email"
+  if ENV["HEROKU_APP_NAME"]
+    OmniAuth.config.full_host = "https://points.ombulabs.com"
+    proxy_to = "points-preview-pr-#{ENV['HEROKU_PR_NUMBER']}.herokuapp.com"
```

Comment:
I did update the preview app url patter so it matches here.

Reply:
Here you can see the new pattern when I hover on the link:
![image](https://user-images.githubusercontent.com/3785596/124583632-9053e080-de53-11eb-9f7b-4f004807af21.png)

Reply:
do we need this specific url pattern? we should be able to use request.host or something like that I think, that way we can reuse this idea for other projects without requiring the change of the url pattern, what do you think? I know we'll need a similar fix for Dash for example